### PR TITLE
Build self contained bundle

### PR DESCRIPTION
**Pull request by fjvallarino**
@tejohnso @ahmedalsudani please review

### CODE REVIEW

**Comment by tejohnso** [comment]
:+1:

File: build-components-packages.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
+CHROME_VERSION=48.0.2564.116
```
Comment (ahmedalsudani) [inline build-components-packages.sh]: @fjvallarino will we have to manually maintain the versions here?

Comment (fjvallarino) [inline build-components-packages.sh]: @ahmedalsudani we will have to manually insert the .ver files until we get rid of them. This script is not actually important, it's something I created to build the new tar.gz we use in the build process from the tar.gz files we use in the current installer.

File: installer/initial_run.bat [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@echo off
+
+md %LOCALAPPDATA%\rvplayer\VERSION
+
+echo Copying files
+robocopy /E /NFL /NDL /NJH /NJS /NC /NS /NP VERSION %LOCALAPPDATA%\rvplayer\VERSION
```
Comment (ahmedalsudani) [inline installer/initial_run.bat]: :laughing: `robocopy`

**Comment by ahmedalsudani** [comment]
:+1:

**Pull request merged**